src/reference/asciidoc/jdbc.adoc

```diff
@@ -378,6 +378,10 @@ The `JdbcChannelMessageStore` implementation provides a more targeted and scalab
 
 Note that you can use a `JdbcMessageStore` to back a message channel, `JdbcChannelMessageStore` is optimized for that purpose.
 
+IMPORTANT: Starting with version 5.1.2, the indexes for the `JdbcChannelMessageStore` have been optimized.
+If you have large message groups in such a store, you may wish to alter the indexes.
+Furthermore, the index for `PriorityChannel` is commented out because it is not needed unless you are using such channels backed by JDBC.
```

Comment:
I think we need also to document that index for the priority must be present in DB  because it is used in the hint in the query hard-coded in the `OracleChannelMessageStoreQueryProvider`.

Reply:
OK; but from my experience; invalid information in a hint is simply ignored.